doc/hacking/using-devcontainers.rst

```diff
@@ -200,6 +200,56 @@ with a database snapshot from a production server. You can read more about
 :ref:`migrating_prod_db_to_dev`.
 
 
+IPv6 connectivity
+=================
+
+NAV is fully capable of working over IPv6, but Docker is not usually configured
+to work with IPv6 out-of-the-box. This presents a challenge if you need to
+communicate with devices over IPv6 while developing inside the
+devcontainer. Fortunately, since `Docker version 27`_, IPv6 has become a lot
+easier to configure.
+
+The NAV devcontainer definition already places all the service containers (as
+defined in :file:`.devcontainer/docker-compose.yml`) on a separate IPv6-enabled
+network. This network is configured with IPv6 masquerading, to ensure that your
+service containers are not exposed on the public IPv6 network.
+
+To enable IPv6 in your Docker daemon, you will need to add something like the
+following stanza to your Docer system's :file:`daemon.json` and restart the
```

Comment:
Would be helpful to mention to create this file if it does not exist

Reply:
Referenced now!